docs/07_admin/01_REST.rst

```diff
@@ -18,5 +18,273 @@
++=====================+========+================================================+=============================+
+| key                 | string | The key that identifies the category. |br|
+|                     |        | If the key already exists as a category |br|
+|                     |        | then the contents of this request |br|
+|                     |        | is merged with the data stored.
++---------------------+-----------+------------------------------------------------+-----------------------------+
+| description         | string | The severity of the event that triggered |br|  | FATAL                       |
+|           |           | the audit trail entry to be written. |br|      |                             |
+|           |           | This will be one of FATAL, ERROR, WARNING |br| |                             |
+|           |           | or INFORMATION.                                |                             |
++---------------------+--------+------------------------------------------------+-----------------------------+
+| items               | array  | A JSON object that describes the detail |br|   | { "message" : |br|          |
++---------------------+-----------+------------------------------------------------+-----------------------------+
+| items[].name        | string | A JSON object that describes the detail |br|   | { "message" : |br|          |
++---------------------+--------+------------------------------------------------+-----------------------------+
+| items[].description | string | A JSON object that describes the detail |br|   | { "message" : |br|          |
++---------------------+--------+------------------------------------------------+-----------------------------+
+| items[].type        | string | A JSON object that describes the detail |br|   | { "message" : |br|          |
++---------------------+--------+------------------------------------------------+-----------------------------+
+| items[].default     | string | A JSON object that describes the detail |br|   | { "message" : |br|          |
++---------------------+--------+------------------------------------------------+-----------------------------+
+
+
+
+
+
+ping
+----
+
+The Ping interface gives a basic confidence check that the FogLAMP appliance is running and the API aspect of the appliance if functional. It is designed to be a simple test that can  be applied by a user or by an HA monitoring system to test the liveness and responsiveness of the system.
+
+
+GET Ping
+~~~~~~~~
+
+``GET /foglamp/ping`` - returns liveness of FogLAMP
+
+
+**Response Payload**
+
+The response payload is the uptime information in a JSON object.
+
++--------+---------+---------------------------------------+---------+
+| Name   | Type    | Description                           | Example |
++========+=========+=======================================+=========+
+| uptime | numeric | Time in seconds since FogLAMP started | 32892   |
```

Comment:
This is now out of date. Ping returns more data than just uptime.

Reply:
This is all I have with the latest version in develop:

foglamp@vbox-dev:/usr/local/foglamp$ curl -s http://localhost:8081/foglamp/ping | jq
{
  "uptime": 50.67945051193237
}